_test/test_IX_classes/test_plot_singleton.m

```diff
@@ -25,391 +31,611 @@
             S.marker_types = {'o'};
             S.maxspec_1D = 1000;
             S.maxspec_2D = 1000;
-            
+
             obj.S_default = orderfields(S);
         end
-        
+
+        function setUp(obj)
+            obj.current_graphics_settings = genieplot.get();
+        end
+        function tearDoan(obj)
```

Comment:
The function name is mispelt: it should be "tearDown". I think this means that the function will not  be run and so the graphics settings will not be reset.